packages/runtime-logger/virtual.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@it-astro:logger:*' {
```

Comment:
Could it be useful to use `addDts` instead of the wildcard for users?

Reply:
I see `addDts` as a tool to provide final users, author of Astro _project_, with custom types based on their use. It doesn't really apply here since this is a tool for integration authors that provides a type for those authors.

The dts wouldn't be generated for them to use in the integration code

Reply:
ah yeah true!